### ISSUE TITLE
Create 'Study' route / view

### ISSUE THREAD

**Issue by NiloCK** [NiloCK/vue-skuilder#16]
The basic view of the application is the `/study` route.

Initial steps:

1. [ ] ~~Basic loop: display a random card, send completed work to the db, and load another random card~~ (Clarified below)
2. [ ] ~~#6 Attach a scheduler to supplement display of random cards~~ (Out of Scope)
3. [ ] ~~Add support for individual users~~ (Out of scope here)

**Comment by NiloCK** [comment]
Infrastructure exists now for questions to pass data up to the `Study` view, but work (#25) needs to be completed so that all questions hook into this infrastructure.

After #25, have a look back here at writing records to the db and working on a visual transition.
